### DISCUSSION TITLE
Consulta sobre Estado Interno de un Objeto

### DISCUSSION THREAD

**Discussion by heinHertz** [dds-jv/foro#112] [Consultas de objetos] [answered]
Hola tengo una duda , no sé si hice bien en ponerlos acá o en el Grupo de Mails de Diseño de Sistemas.
1)  En varios Apuntes hablan de Estado Interno de un Objeto, no me queda claro ese concepto, es algo que tiene que ver con que un objeto conozca a un solo Objeto , que tenga un Atributo especial?

**Comment by gastonprieto** [comment, marked as answer]
En el mail ninguna consulta :yum: Esta muy bien que sea acá :+1: 

El estado interno de un objeto esta dato por sus atributos y a quienes estan referenciando.
